repos/system_upgrade/el7toel8/actors/persistentnetnamesconfig/actor.py

```diff
@@ -70,7 +47,7 @@ def process(self):
 
                     # TODO(msekleta): we need to introduce another actor which will
                     # # install generated link files to RHEL-8 initrd image
```

Comment:
@drehak pls remove the todo msg. It's already resolved (see the line producing the `InitrdIncludes` msg